src/utils/log.go

```diff
@@ -6,24 +6,50 @@ import (
 	"fmt"
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+	"io"
 	"os"
+	"sync"
 )
 
 type Writer struct {
-	writer *bufio.Writer
-	buffer *bytes.Buffer
+	mutex   *sync.Mutex
+	writers []io.Writer
+	buffer  *bytes.Buffer
+}
+
+// ConnectTo allows write to multiple targets
+func (w *Writer) ConnectTo(writer io.Writer) {
+	w.mutex.Lock()
+	defer w.mutex.Unlock()
+	w.writers = append(w.writers, writer)
 }
 
 func (w *Writer) Write(p []byte) (n int, err error) {
-	return w.writer.Write(p)
+	w.mutex.Lock()
```

Comment:
Logy sa zapisuju z viacerych vlakien, tak sa musel pridat zamok. V pripade suboru to je poriesene, ale v pripade testovacieho logera to nebolo (kedze zapisuje iba do pamate).